src/utils/graph/newgraph.js

```diff
@@ -0,0 +1,192 @@
+import { offsetNode, offsetEdge } from './common';
+import { layout } from './layout';
+import { routing } from './routing';
+
+const defaultOptions = {
```

Comment:
Would you mind changing this filename to just `graph.js` and that way it will be inline with the test filename too?

Reply:
Sure, done